lib/dor/assembly/stub_content_metadata_parser.rb

```diff
@@ -12,8 +12,6 @@ def convert_stub_content_metadata
         # uses the assembly-objectfile gem to create content metadata using the stub contentMetadata provided
         load_stub_content_metadata
 
-        LyberCore::Log.info("Creating content metadata from stub for #{druid.id}")
```

Comment:
Because logging is now provided by Sidekiq, this would have required passing in the logger which would have meant more code changes which didn't seem worth it for a logging comment that was largely redundant of the logging made by the robot itself.